demos/ag-alert-keyvault-advisor/azuredeploy.json

```diff
@@ -0,0 +1,64 @@
+{
+   "$schema":"https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "alertName":{
+         "type":"String"
+      },
+      "applicationGatewayResourceGroup":{
+         "type":"String",
+         "metadata":{
+            "description":"Resource Group example /subscriptions/subs-id/resourceGroups/rg-name"

```

Comment:
Resource group's resource ID seems to be required here, but the Prereq is populating Application Gateway resource ID